_data/other.yml

```diff
@@ -343,6 +343,13 @@ websites:
       tfa: Yes
       software: Yes
 
+    - name: Ulule
+      url: https://www.ulule.com/
+      twitter: ulule
+      facebook: ulule
```

Comment:
Their Facebook page redirects to `ululeEN` for me. Does anyone else get a different result after going to [facebook.com/ulule](https://www.facebook.com/ulule)?